react-ui/src/App.js

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react'
 import LogIn from "./components/Layouts/LogIn"
 import Dashboard from "./components/Dashboard/Dashboard"
 import Signup from "./components/Layouts/Signup"
+import WorkerInputpage from './components/Layouts/InputWorkerTimes/WorkerInputpage'
```

Comment:
Hi Kyle,
This import statement is no longer needed. I would recommend removing it to reduce future confusion.

Reply:
Removed import statement